.dockerignore

```diff
@@ -0,0 +1,2 @@
+.dockercache
```

Comment:
```suggestion
# During build .dockercache and .gocache download locations used to speed up the build
# as these are large they are ignored to avoid the build copying them into docker context
.dockercache
```

Quick comment to explain why I made this change